src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,57 @@
+package tasks;
+
+/**
+ * Task class that handles a task and methods that manipulated the task
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * A Method to mark the task as completed
+     */
+    public void taskDone() {
+        this.isDone = true;
+    }
+
+    /**
+     * A method to get the status icon
+     * 
+     * @return the symbol that shows if an event is done or not
+     */
+    public String getStatusIcon() {
+        return (isDone ? "X" : " "); // mark done task with X
```

Comment:
Are the parentheses necessary here? I do not think it is needed for the coding standard.